CMakeLists.txt

```diff
@@ -221,27 +226,29 @@ set(NMC_VALIDATION_DATA_DIR "${PROJECT_SOURCE_DIR}/validation/data" CACHE PATH
   "location of generated validation data")
 
 #----------------------------------------------------------
-# Whether to build validation data at all
+# Whether to build validation data
```

Comment:
I have a strong opposition to just storing pre-generated data in the repository. The discussion continues.